easybuild/easyconfigs/n/NextPolish/NextPolish-1.4.1-GCC-13.3.02025-01-08.eb

```diff
@@ -0,0 +1,100 @@
+easyblock = 'MakeCp'
```

Comment:
I think you added the new file to the repo to fix the previous review, but did not delete this one